### ISSUE TITLE
Sample rate, bit depth, channels, HD audio

### ISSUE THREAD

**Issue by certuna** [navidrome/navidrome#1438]
This is a placeholder issue for support for various audio characteristics & how to display them in the UI. There's been plenty of discussion on Discord but no centralized place for implementations.

**Stream info**
- [x] `bit_rate`
- [x] `channels` https://github.com/navidrome/navidrome/pull/1294
- [ ] `sample_rate`
- [ ] `bit_depth`
- [ ] `codec`
- [ ] `modulation` : PCM or DSD?
- [ ] `hd` : boolean constructed from the values above: true if sample_rate > 44.1k or bit_depth > 16 or modulation = DSD?
**File attributes**
- [x] `suffix` (file extension)
**Tag fields**
- [ ] `TSSE` "Software/Hardware and settings used for encoding"
- [ ] `TFLT` "File type"
- [ ] `TMED` "Media type"
- [ ] `TENC` "Encoded by"

**Comment by certuna** [comment]
still relevant

**Comment by certuna** [comment]
still relevant

**Comment by metalheim** [comment]
not stale!
Some if it will be adressed in the BFR #2709 like `BitDepth` and `SampleRate`

**Comment by emes81** [comment]
I would heavily support this, especially for labelling FLAC quality. A few years ago, 44/16 was the default, but now FLACs are coming in various bitrates and depths.

**Comment by ethanuppal** [comment]
Yeah, it would be great to see what quality the audio is streaming as. Very surprised this isn't a feature yet.
